stages/org.osbuild.anaconda

```diff
@@ -5,8 +5,8 @@ Configure basic aspects of the anaconda installer
 Create an anaconda configuration file `90-osbuild.conf` in
 the folder `/etc/anaconda/conf.d` to configure anaconda.
 
-Currently only the list of enabled kickstart modules is
-configurable via the `kickstart-modules` option.
+Can configure the list of enabled `kickstart_modules`, `activatable_modules`,
+`forbidden_modules`, and `optional_modules`.
```

Comment:
Just drop that paragraph entirely? I mean, the schema conveys the same information, doesn't it?

Reply:
👍 Yeah, we used to do it (describing the options in the full text), but I also think it is not necessary.

Reply:
Ack, gone.